src/Node/Buffer.purs

```diff
@@ -81,8 +92,27 @@ usingFromImmutable f buf = f <$> unsafeFreeze buf
 usingToImmutable :: forall a. (a -> ImmutableBuffer) -> a -> Effect Buffer
 usingToImmutable f x = unsafeThaw $ f x
 
+-- | Creates a new buffer of the specified size. Alias to `alloc`.
 create :: Int -> Effect Buffer
-create = usingToImmutable Immutable.create
+create = alloc
+
+-- | Creates a new buffer of the specified size.
+alloc :: Int -> Effect Buffer
+alloc = usingToImmutable Immutable.alloc
+
+-- | Creates a new buffer of the specified size. Unsafe because it reuses memory from a pool
+-- | and may contain sensitive data. See the Node docs.
```

Comment:
Worth providing a link to the Node docs for this, or is there not an easily-linkable section (same for the rest of the file where you've commented about Node docs)?

Reply:
Done.